packages/linter/src/rules/NoIconFontIsUsed.ts

```diff
@@ -0,0 +1,103 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+const postcss = require("postcss");
+const safeParser = require("postcss-safe-parser");
+
+const ICON_FONT_IDENTIFIERS = [
+  {
+    className: "fa-",
```

Comment:
Do we even need to match by classname? Or is it common that icon font names are changed?

Reply:
As mentioned above, there are some cases that use an external stylesheet via Google Web Fonts, and a way to identify those, would be to match by classname.